src/components/Image.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import Img from 'next/image';
+
+const Image = (props) => {
+
+    const [error, setError] = useState(false);
+    
+    const {
+        src,
+        width = 240,
+        height = 240,
+        alt = '',
+        ...otherProps
+    } = props;
+    
+    // URL to use if the actual image fails to load.
+    const fallBackUrl = `https://via.placeholder.com/${width}x${height}?text=${alt}`;
+
+    /**
+     * Handles any error when loading the image.
+     * 
+     * @return {void}
+     */
+    const errorHandler = () => {
+        setError(true);
+    }
+
+    return (
+        <>
+            <Img 
+            src={error ? fallBackUrl : src}
```

Comment:
Can we use imageSizes and deviceSizes properties.
https://nextjs.org/docs/api-reference/next/image#other-props
https://nextjs.org/docs/basic-features/image-optimization#device-sizes

Reply:
My concern was having the ability to override the defaults, if any of the component passes those values.
It's fine for now.  Let's continue with what we currently have.
Adding this link here for future discussion
https://github.com/vercel/next.js/issues/18413#issuecomment-721747908